src/Serializer/AbstractSerializer.php

```diff
@@ -357,4 +383,21 @@ public function getSerializeAllObjects(): bool
     {
         return $this->serializeAllObjects;
     }
+
+    /**
+     * @param mixed $callable
+     *
+     * @throws InvalidArgumentException
+     */
+    private function assertIsValidCallable($callable): void
```

Comment:
I don't see any good reason to have (in this case) a `private` method that just checks whether the argument is a callable and throws if it's not. Please inline the code where it belongs to